mmdet3d/datasets/nuscenes_dataset.py

```diff
@@ -267,10 +267,14 @@ def get_data_info(self, index: int) -> Dict[str, Any]:
                 camera2ego[:3, 3] = camera_info["sensor2ego_translation"]
                 data["camera2ego"].append(camera2ego)
 
-        # TODO (Haotian): test set submission.
+                # camera to lidar transform
+                camera2lidar = np.eye(4).astype(np.float32)
+                camera2lidar[:3, :3] = camera_info["sensor2lidar_rotation"]
+                camera2lidar[:3, 3] = camera_info["sensor2lidar_translation"]
+                data["camera2lidar"].append(camera2lidar)
```

Comment:
Could you double-check whether the other transformations are still actively being used? If not, we could remove them.

Reply:
Sure, we can remove this in a future version with coordinate system reformatting.